### ISSUE TITLE
TabView scroll bar is too big on WASM

### ISSUE THREAD

**Issue by MartinZikmund** [unoplatform/uno#4723]
## Current behavior

When `TabView` needs a scroll bar on WASM, it uses the big wide horizontal scroll bar style, which covers most of the tabs and they even move up when scrolled.

## Expected behavior

Scroll bar should not cover the tabs and should auto hide, like on UWP

## How to reproduce it (as minimally and precisely as possible)

Go to `TabViewPage` sample on WASM Samples app and add tabs until scroll bar appears.

## Environment

Nuget Package:

- [x] Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia
- [ ] Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia
- [ ] Uno.SourceGenerationTasks
- [ ] Uno.UI.RemoteControl / Uno.WinUI.RemoteControl
- [ ] Other: 

Nuget Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [x] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] macOS
- [ ] Skia
  - [ ] WPF
  - [ ] GTK (Linux)
  - [ ] Tizen
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

**Comment by dr1rrb** [comment]
This has most probably been fixed by latest updates

**Comment by MartinZikmund** [comment]
@dr1rrb currently it seems the scroll bar does not show up at all for `TabView`, which is the correct behavior, but the mouse wheel does not work there unfortunately

**Comment by dr1rrb** [comment]
@MartinZikmund actually I'm able to scroll using the horizontal wheel of my mouse ... but I would imagine that on UWP you are able to scroll using the vertical one?

**Comment by MartinZikmund** [comment]
@dr1rrb Yes, that's true. I think that when there is just horizontal scroll viewer (not nested in a vertical), it responds to vertical mouse wheel too.

**Comment by dr1rrb** [comment]
On UWP yes ... but I tested with a very complex HTML page, it's not the case in browsers:

```html
<html>
<body>
<div style="width: 100px; height: 100px; background: green; overflow: scroll">
  <div style="width: 1024px; height: 60px; background: red;">
    Hello world
  </div>
</div>
</body>
</html>
```

This means that we need to add explicit support for it in uno (and not only for `TabView`: going to change the title of the issue)

**Comment by MartinZikmund** [comment]
This is not a problem in Skia WASM
